FRENCH/src/ch07-03-paths-for-referring-to-an-item-in-the-module-tree.md

```diff
@@ -0,0 +1,883 @@
+-->
+
+Dans le chemin absolu, nous commençons avec `crate`, la racine de notre
+arborescence de modules de notre crate. Ensuite, le module `salle_a_manger` est
+défini dans la crate racine. Le module `salle_a_manger` n'est pas public, mais
+comme la fonction `manger_au_restaurant` est définie dans le même module que
+`salle_a_manger` (car `manger_au_restaurant` et `salle_a_manger` sont frères),
+nous pouvons utiliser `salle_a_manger` à partir de `manger_au_restaurant`.
+Ensuite, nous avons le module `accueil`, défini avec `pub`. Nous pouvons accéder
+au module parent de `accueil`, donc nous pouvons accéder à `accueil`. Enfin, la
+fonction `ajouter_a_la_liste_attente` est elle aussi définie avec `pub` et nous
+pouvons accéder à son module parent, donc au final cet appel à la fonction
+fonctionne bien !
+
+<!--
+In the relative path, the logic is the same as the absolute path except for the
+first step: rather than starting from the crate root, the path starts from
+`front_of_house`. The `front_of_house` module is defined within the same module
+as `eat_at_restaurant`, so the relative path starting from the module in which
+`eat_at_restaurant` is defined works. Then, because `hosting` and
+`add_to_waitlist` are marked with `pub`, the rest of the path works, and this
+function call is valid!
+-->
+
+Dans le chemin relatif, le fonctionnement est le même que le chemin absolu sauf
+pour la première étape : plutôt que de démarrer de la crate racine, le chemin
+commence à partir de `salle_a_manger`. Le module `salle_a_manger` est défini
+dans le même module que `manger_au_restaurant`, donc le chemin relatif qui
+commence à partir du module où est défini `manger_au_restaurant` fonctionne
+bien. Ensuite, comme `accueil` et `ajouter_a_la_liste_attente` sont définis avec
+`pub`, le reste du chemin fonctionne, et cet appel à la fonction est donc
+valide !
+
+<!--
+### Starting Relative Paths with `super`
+-->
+
+### Commencer les chemins relatifs avec `super`
+
+<!--
+We can also construct relative paths that begin in the parent module by using
+`super` at the start of the path. This is like starting a filesystem path with
+the `..` syntax. Why would we want to do this?
+-->
+
+Nous pouvons aussi créer des chemins relatifs qui commencent à partir du module
+parent en utilisant `super` au début du chemin. C'est comme débuter un chemin
+dans le système de fichiers avec la syntaxe `..`. Mais pourquoi voudrions-nous
+faire cela ?
+
+<!--
+Consider the code in Listing 7-8 that models the situation in which a chef
+fixes an incorrect order and personally brings it out to the customer. The
+function `fix_incorrect_order` calls the function `serve_order` by specifying
+the path to `serve_order` starting with `super`:
+-->
+
+Imaginons le code dans l'encart 7-8 qui représente le cas où le chef prépare une
+commande erronée et l'apporte personnellement au client. La fonction
+`preparer_commande_erronee` appelle la fonction `servir_commande` en commençant
```

Comment:
I'd rather call this function another way to reflect that we are not preparing an incorrect order, but rather *fixing* it, as stated in the original version
```suggestion
`corriger_commande_erronee` appelle la fonction `servir_commande` en commençant
```